index.html

```diff
@@ -183,9 +218,10 @@ <h3>Skills</h3>
         </ul>
         <img src="./Media/ic_skills.svg" alt=" " class="about-img">
       </div>
+      <img class="deco-7" src="./Media/image_dots_with_teal.svg" alt=" ">
     </div>
   </section>
-  <section class="contact-me">
+  <section id="contact" class="contact-me">
```

Comment:
Please ignore this part of the design template and use only one input for the full name for all versions.
![wrong](https://user-images.githubusercontent.com/58376020/125847834-f67f367d-a465-4087-a82f-c79a4226268e.png)

Reply:
Template 5 requires two inputs, name and email. Kept both inputs in the same row just like in the template but reduced the distance between them.